fedoragsearch-transforms/islandora_transforms/slurp_all_MODS_to_solr.xslt

```diff
@@ -73,6 +73,13 @@
     </field>
   </xsl:template>
 
+  <!-- the following template adds a date_of_award field -->
```

Comment:
This is *the* _most_ ridiculous change request yet, but please change the comment :) 

`... adds a cross-collection publication type field` or something.

Reply:
I wouldn't call that ridiculous.  If we really want these comments in there, they should represent what the template match is doing.